semantic-conventions/src/opentelemetry/semconv/templating/markdown/__init__.py

```diff
@@ -414,15 +434,28 @@ def _render_group(self, semconv, parameters, output):
         self.render_ctx.is_remove_constraint = "remove_constraints" in parameters
         self.render_ctx.group_key = parameters.get("tag")
         self.render_ctx.is_full = "full" in parameters
+        self.render_ctx.is_metric_table = "metric_table" in parameters
 
         if isinstance(semconv, EventSemanticConvention):
             output.write("The event name MUST be `{}`.\n\n".format(semconv.name))
 
+        if self.render_ctx.is_metric_table:
+            if isinstance(semconv, MetricSemanticConvention):
+                self.to_markdown_metric_table(semconv, output)
+            else:
+                raise ValueError(
+                    "semconv `{}` was specified with `metric_table`, but it is not a metric convention".format(
+                        semconv.semconv_id
+                    )
+                )
+
         attr_to_print = []
         attr: SemanticAttribute
         for attr in sorted(
             semconv.attributes, key=lambda a: "" if a.ref is None else a.ref
         ):
+            if self.render_ctx.is_metric_table:
```

Comment:
That's a bit of a hack...
Also seeing `remove_constraints` used always, I think it would make sense to separate metric_tables more from all the others. All the others are attribute tables, which the metric_table is not.

I'm thinking about something like `if "metric_table" in parameters: render_metric_group(...); else: render_attribute_group(...)`

Reply:
Yeah, agreed that's a better way. I've done what you've suggested, there's an if/else for rendering metrics vs attributes tables.